### DISCUSSION TITLE
How does one get recognised as a contributor?

### DISCUSSION THREAD

**Discussion by lcgeneralprojects** [py-pdf/fpdf2#1206] [Q&A] [answered]
As the title said.

I can't seem to find what criteria I have to fulfil to be considered a contributor. I have already made a successful PR (which includes new tests) and submitted an issue report, so it seems that I have simply been missed by mistake so far.

If I have been passed over by mistake, I'd like to ask to be credited.

**Comment by Lucas-C** [comment, marked as answer]
Hi @lcgeneralprojects 

Thank you for asking.
Indeed, we simply forgot to add you as a contributor, please excuse use for that omission 😅

I have just fixed that in PR #1208 
Thank you for your contributions! 🙏
